mpu6050_scripts/get_temperature.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#Init
+sudo i2cset -y 0 0x68 0x6B 0
+
+while true
+do
+
+#Read data
+UB=$(sudo i2cget -y 0 0x68 0x41)
+LB=$(sudo i2cget -y 0 0x68 0x42)
+
+#Upper Bit + Lower Bit
+TEMP=$(( (UB<<8)+LB-65535 ))
```

Comment:
Is it require to use -65535?

Reply:
Yes, this is the only way to XOR all 16 bits. I don't know why, but ^0xFFFF works incorrectly in the bash.

Reply:
Why do You need XOR for all bits?